resources/user-agents/browsers/safari/safari-15-0-to-15-4.json

```diff
@@ -1,6 +1,6 @@
 {
     "division": "Safari #MAJORVER#.#MINORVER#",
-    "versions": ["15.4", "15.3", "15.2", "15.1", 15],
+    "versions": ["15.5", "15.4", "15.3", "15.2", "15.1", 15],
```

Comment:
Do we have any example UA strings for these?

Reply:
just pushed